Sources/NIOTransportServices/NIOTSChannelOptions.swift

```diff
@@ -21,6 +22,19 @@ public struct NIOTSChannelOptions {
     public static let waitForActivity = NIOTSChannelOptions.Types.NIOTSWaitForActivityOption()
 
     public static let enablePeerToPeer = NIOTSChannelOptions.Types.NIOTSEnablePeerToPeerOption()
+    
+    public static let currentPath = Types.NIOTSCurrentPathOption()
+
+    @available(OSX 10.15, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
```

Comment:
This guard (and associated guards) is wrong: the metadata accessor was available in macOS 10.14 and watchOS 5.